docs/chapters/04_features.md

```diff
@@ -250,18 +282,20 @@ While it is OK to know how to manually send GraphQL request, you normally don't
 
 To have a great developer experience, we **strongly recommend** to use a GraphQL client for your platform of choice. Here are some great ones:
 
-- **[Hoppscotch (formerly Postwoman)](https://hoppscotch.io/)**: Ideal for testing sending manual requests, getting the schema, etc.
+- **[Altair](https://altair.sirmuel.design/)**: Ideal for testing sending manual requests, getting the schema, etc.
```

Comment:
Switched to Altair, because it is the easiest GraphQL client that allows testing localhost

Reply:
Does Altair support adding headers on WebSockets? Because then we could also close #656

Reply:
If by headers you mean connection params, then yes!

Reply:
I don't think the user agents support specifying headers as part of a web socket upgrade request, and that's why the connection params was added as part of the graphql subscriptions over websocket specification, to be able to authenticate the subscription requests AFAIK. I detailed a bit more about it in this issue: https://github.com/imolorhe/altair/issues/1419#issuecomment-759148821